### PR TITLE
Inventory - Improve multiple aspects

### PR DESCRIPTION
**Pull request by johnb432**
**When merged this pull request will:**
- Moved name-weight bar modification from `ace_movement` into `ace_inventory`.
- Improved medical item detection.
- Improved item filtering:
    - Now removes medical and face wear items from 'items' tab, as they have their own tabs (face wear is in the head gear tab).
    - Fixed an issue with a bad macro, making vests pop up in the 'items' tab when they shouldn't.
- Closes https://github.com/acemod/ACE3/issues/7150.

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by LinkIsGrim** [comment]
Medical should remain in item filter

**Comment by johnb432** [comment]
> Medical should remain in item filter

You want medical items to remain the items tab, despite there being a dedicated tab for them? Feels unnecessary.

**Comment by LinkIsGrim** [comment]
> > Medical should remain in item filter
> 
> You want medical items to remain the items tab, despite there being a dedicated tab for them? Feels unnecessary.
> I added some clarification in the post.

Yeap

File: addons/inventory/initSettings.inc.sqf [hunk 1]
```diff
@@ -1,9 +1,8 @@
-private _category = [LELSTRING(common,categoryUncategorized), localize "str_a3_gear1"];
-
 [
-    QGVAR(inventoryDisplaySize), "LIST",
+    QGVAR(inventoryDisplaySize),
+    "LIST",
     [LSTRING(SettingName), LSTRING(SettingDescription)],
-    _category,
+    [LELSTRING(common,categoryUncategorized), localize "str_a3_gear1"],
```
Comment (LinkIsGrim) [inline addons/inventory/initSettings.inc.sqf:5]: I prefer _category for readability, even if there's only one setting

**Review by LinkIsGrim — COMMENTED** [review]
Everything else LGTM

**Comment by LinkIsGrim** [comment]
> Medical should remain in item filter
Fuck it, we'll just wait till someone complains about it

**Review by LinkIsGrim — APPROVED** [review]

**Comment by jonpas** [comment]
Medical items showing only in medical tab makes sense to me.

**Comment by rautamiekka** [comment]
> > > Medical should remain in item filter
> > 
> > 
> > You want medical items to remain the items tab, despite there being a dedicated tab for them? Feels unnecessary.
> > I added some clarification in the post.
> 
> Yeap

Weirdly arbitrary, ngl.

> Medical items showing only in medical tab makes sense to me.

I mean, that's the whole point. 100% agree.

**Pull request merged**